src/manifests/test_manifest.py

```diff
@@ -62,6 +62,19 @@ class TestManifest(ComponentManifest['TestManifest', 'TestComponents']):
                         "type": "dict",
                         "schema": {
                             "build-dependencies": {"type": "list"},
+                            "topology": {
+                                "type": "list", "required": False,
+                                "schema": {
+                                    "type": "dict",
+                                    "schema": {
+                                        "cluster_name": {"required": True, "type": "string"},
+                                        "data_nodes": {"type": "integer", "required": True},
+                                        "cluster_manager_nodes": {"type": "integer"}
+                                    }
+
+                                }
+
+                            },
```

Comment:
Does this support older version of test manifest?
Could you provide example of the new manifest as part of the pytest?
Thanks

Reply:
Yes older version of test_manifest is supported, the Topology config is optional and defaults to one data node is not configured.